tests/mutaterows_test.go

```diff
@@ -336,17 +336,18 @@ func TestMutateRows_Retry_ExponentialBackoff(t *testing.T) {
 
 	// 4b. Log the retry delays
 	origReq := <-recorder
-	firstRetry := <-recorder
-	secondRetry := <-recorder
-	thirdRetry := <-recorder
```

Comment:
This code was hanging indefinitely when the client made fewer than three retries.